CMakeLists.txt

```diff
@@ -47,6 +47,67 @@ if(POLICY CMP0135)
   cmake_policy(SET CMP0135 NEW)
 endif()
 
+
+if(WIN32)
+    if(DEFINED ENV{BUILDENV_BASEPATH})
+        set(BUILDENV_BASEPATH "$ENV{BUILDENV_BASEPATH}\\")
```

Comment:
In other places we ignore environment variables if they are explicit passed to cmake via -D. 
I think we should follow the same priority here and below with BUILDENV_URL